loopd/config.go

```diff
@@ -152,6 +151,7 @@ type Config struct {
 	TLSExtraDomains    []string `long:"tlsextradomain" description:"Adds an extra domain to the generated certificate."`
 	TLSAutoRefresh     bool     `long:"tlsautorefresh" description:"Re-generate TLS certificate and key if the IPs or domains are changed."`
 	TLSDisableAutofill bool     `long:"tlsdisableautofill" description:"Do not include the interface IPs or the system hostname in TLS certificate, use first --tlsextradomain as Common Name instead, if set."`
+	TLSValidity        int      `long:"tlsvalidity" description:"Loop's TLS certificate validity period in days. Defaults to 430 (roughly 14 months)"`
```

Comment:
nit: Alternatively this could be a `time.Duration` variable so could be configured as:
```tlsvalidity=365d```

Reply:
So? moving to `time.Duration` and `365d` as other comments suggested?